includes/classes/SiteAutomation/Request.php

```diff
@@ -78,10 +78,24 @@ public function get( $page, $widget ) {
 		$this->widget  = $widget;
 		$this->api_url = $this->set_api_url( $page, $widget );
 
-		$this->status = $this->get_status();
+		$this->status         = $this->get_status();
 		$site_automation_data = false;
 
-		if ( ! defined( 'SOPHI_BYPASS_CACHE' ) || ! SOPHI_BYPASS_CACHE ) {
+		/**
+		 * Whether to bypass caching.
+		 *
+		 * @since 1.0.9
+		 * @hook sophi_bypass_get_cache
+		 *
+		 * @param {bool} $bypass_cache True or false.
+		 * @param {bool} $page Page name.
+		 * @param {bool} $widget Widget name.
```

Comment:
@oscarssanchez the last two parameters are specified as bool, they should be string. (Nitpicking: parameter descriptions should be aligned)

Also, correct me if I'm wrong, I believe phpdoc does not require your parameter type inside of { ...} curly brackets (that's a jsdoc convention).

Reply:
(The same goes for the 'sophi_curated_post_list' filter doc in Integration.php)